src/main/cpp/mechanisms/ClimberManager/ManualClimbState.cpp

```diff
@@ -58,6 +58,8 @@ void ManualClimbState::InitPRACTICE_BOT9999()
 void ManualClimbState::Run()
 {
 	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("ManualClimbState"), string("Run"));
+	units::angle::degree_t TargetChange = units::angle::degree_t(TeleopControl::GetInstance()->GetAxisValue(TeleopControlFunctions::MANUAL_CLIMB)*m_manualClimbRate);
+	m_manualTarget += TargetChange;
 }
```

Comment:
Need to Call UpdateClimberTargetMethod and send the target to the motor
Also you should use the clamp method to clamp the value we can send the motor so we don't break the climber. for now set it to -10 to 95 deg

Reply:
Also make the max and min member variables